lib/scripts/job_runs.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Why `lib/scripts` instead of the root level `bin` or `deploy` folder?

Reply:
I moved it to the /bin/ folder. 

I had no reason other than it seemed like the logical place. The task its related to lives in /lib/tasks.